src/Sylius/Component/Payment/composer.json

```
@@ -55,7 +56,7 @@
     },
     "autoload-dev": {
         "psr-4": {
-            "Sylius\\Component\\Payment\\spec\\": "spec/"
+            "Tests\\Sylius\\Component\\Payment\\": "Tests/"
```

Use lowercase for `tests` directory name
```suggestion
            "Tests\\Sylius\\Component\\Payment\\": "tests/"
```